libs/design-system/src/pagination/Pagination.tsx

```diff
@@ -0,0 +1,97 @@
+import styled from '@emotion/styled';
+import { PropsWithChildren } from 'react';
+import { clamp } from '../utils';
+import { colors } from '../config';
+import { mantineConfig } from '../config/theme.config';
+import { ControlBar } from './ControlBar';
+import { ControlButton } from './ControlButton';
+import { GoToPageInput } from './GoToPageInput';
+import { PageSizeSelect } from './PageSizeSelect';
+import { FIRST_PAGE_NUMBER } from './Pagination.const';
+import { IPaginationContext, PaginationContext } from './PaginationContext';
+import { clampPageNumber } from './util';
+
+const PaginationWrapper = styled.div(
+  ({ theme }) => `
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+
+    & input {
+      margin: 0;
+      min-height: inherit;
+      height: 32px;
+      min-width: 44px;
+      font-weight: 700;
+    } 
```

Comment:
I preferred this way to give consistent styling to both types of inputs (where possible)

Reply:
I see what you are saying. I don't have a strong opinion, but my feelings are that because of the composition, it's better to have it tied with the related place. 

For example, I might want to use the custom component `1 of 20 pages` (not a good example but anyway), where `1` will be in the input, and I might not necessarily want to have input `min-width: 44px`, then I would have to have a higher CSS specificity to be able to override that value.

Also, I wouldn't expect to look at the pagination component for the styles of the `GoToPageInput` component.

Reply:
Okay sounds good, I will switch it